src/Wallabag/CoreBundle/Resources/translations/messages.de.yml

```diff
@@ -297,12 +297,12 @@ quickstart:
         instapaper: 'von Instapaper migrieren'
     developer:
         title: 'Entwickler'
-        # description: 'We also thought to the developers: Docker, API, translations, etc.'
+        description: 'Wir haben auch an die Entwickler gedacht: Docker, API, Übersetzungen, etc.'
         create_application: 'Erstelle eine Anwendung und nutze die wallabag API'
-        # use_docker: 'Use Docker to install wallabag'
+        use_docker: 'Nutze Docker, um wallabag zu installieren'
     docs:
         title: 'Komplette Dokumentation'
-        # description: "There are so much features in wallabag. Don't hesitate to read the manual to know them and to learn how to use them."
+        description: "Es gibt viele Features in wallabag. Scheue dich nicht die Anleitung zu lesen, um sie kennenzulernen und zu lernen sie zu nutzen."
```

Comment:
I'm not 100% happy with the translation of the second sentence. Sounds too complicated for me. Maybe @mruminski has an idea?